app/src/main/java/com/infomaniak/mail/ui/main/menu/MoveFragment.kt

```diff
@@ -165,14 +162,41 @@ class MoveFragment : MenuFoldersFragment() {
         }
     }
 
-    private fun toggleFolderListsVisibility(isSearching: Boolean) = with(binding) {
-        searchResultsList.isVisible = isSearching
-        moveFoldersLists.isGone = isSearching
+    private fun toggleFolderListsVisibility(query: String?, allFolders: List<Folder>) {
+        isSearching = !query.isNullOrBlank()
+        if (!isSearching) folderAdapter.setFolders(allFolders, moveViewModel.currentFolderId)
```

Comment:
If we use a single live data we don't need this anymore, but as of now, the name of the method doesn't mean anything anymore. We're not toggling the visibility of two lists anymore

Reply:
We don't want to use a single liveData.

Reply:
Fixed here : https://github.com/Infomaniak/android-kMail/pull/1901